CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Feature Requests
+Want a new feature in Mudlet? Let us know about it! We also have a long-running feature requests thread on [our forums](http://forums.mudlet.org/viewtopic.php?f=5&t=92).
+
+# Bug Reports
+We're sorry you've encountered an issue with Mudlet and appreciate that you're taking the time to report it to us. 
+
+To help us find & fix the problem quickest, please include:
+* as much information possible about the issue
+* your Mudlet version as it says in the ``About`` window
+* specific instructions on how to reproduce the issue if you know how to
+
+# Coding guidelines
+
+If you're a first-timer, you're excluded, we'll go easy on you :wink:
+
+## Use QLatin1String over QStringLiteral if possible
```

Comment:
Oh __bottom__ I wasn't aware of that - guess it must be the thing I count as some I learnt today.